database/migrations/2016_06_29_205018_add_oauth_fields_to_users.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Yep, these are the two bundled migrations with Northstar PHP, for storing user access/refresh tokens and client access tokens respectively. They're not used unless authenticating via Northstar, so safe to remove for now.